pytrip/__init__.py

```diff
@@ -24,6 +24,7 @@
 # flake8: noqa
 
 import logging
+from pytrip.cube import Cube
```

Comment:
Sure, user can decide but will need to use numpy method:

With subclassed he could do:
```c = CtxCube(...)
c.write('blabla')
```

With master class either:
```
c = Cube(...)
c.write_trip_header('blabla.some_ext1')
c.write_trip_data('blabla.some_ext2')
```

or pickle or some direct method.

Reading from trip format (`read_trip_data_file` method in Cube class) also will not work, as it needs to figure out subclass based on extension.